### ISSUE TITLE
Command line fullscreen switch

### ISSUE THREAD

**Issue by AlexanderRevo** [0ldsk00l/nestopia#361]
Hi, can we have a command line option to start Nestopia in fullscreen, something like `--fullscreen`?

It would be useful for creating a shortcut to just run a specific game in fullscreen, but keeping Nestopia starting in windowed mode when running it without a game loaded, which AFAIK isn't something that can be achieved purely though configuration options.

Many other emulators support this option and it would be great to have in it Nestopia.

Aslo, I've found some old documentation at https://mhxx43.tripod.com/readme.html, but the provided example doesn't seem to work.

**Comment by rdanbrook** [comment]
Do you mean for the win32 build? The FLTK build has `-f` and `--fullscreen` already. You said shortcut so I assume you're talking about win32. It shouldn't be a big deal to add this.

**Comment by AlexanderRevo** [comment]
Yes, sorry, forgot to mention the platform. It's the Win32 version, indeed :)
